src/runner/worker.rs

```diff
@@ -59,15 +57,13 @@ impl<'a, DB: WriteResults + Sync> Worker<'a, DB> {
 
     pub(super) fn run(&self) -> Fallible<()> {
         // This uses a `loop` instead of a `while let` to avoid locking the graph too much
+        let mut guard = self.graph.lock().unwrap();
         loop {
             self.maybe_cleanup_target_dir()?;
-            let walk_result = self
-                .graph
-                .lock()
-                .unwrap()
-                .next_task(self.ex, self.db, &self.name);
+            let walk_result = guard.next_task(self.ex, self.db, &self.name);
             match walk_result {
                 WalkResult::Task(id, task) => {
+                    std::mem::drop(guard);
```

Comment:
Why the full qualification instead of just `drop(guard)`?

Reply:
It's in the prelude? I can never remember that. Seems fine regardless.

Reply:
Yep, it's in the prelude: https://doc.rust-lang.org/nightly/std/prelude/index.html#prelude-contents